doc/user-guides/anonymous-access.md

```diff
@@ -0,0 +1,177 @@
+# Enforcing anonymous access with Kuadrant AuthPolicy
+
+Learn how to allow anonymous access to certain endpoints using Kuadrant's `AuthPolicy`
+
+## Requisites
+
+- [Docker](https://docker.io)
+
+## Run the guide ① → ④
+
+### ①  Setup 
+
+Clone the repo:
+
+```sh
+git clone git@github.com:Kuadrant/kuadrant-operator.git && cd kuadrant-operator
+```
+
+Run the following command to create a local Kubernetes cluster with [Kind](https://kind.sigs.k8s.io/), install & deploy Kuadrant:
+
+```sh
+make local-setup
+```
```

Comment:
Yeah, this may be overkill, but currently aligns with the other guides. Overall, I agree, we should probably update the guides to helm install kuadrant and other components to decouple the user guides from the development workflow

Reply:
I like the solution of stating the prerequisite _"You have installed Kuadrant..."_

Reply:
Yes, same. I took it from https://github.com/Kuadrant/kuadrant-operator/pull/1054/files#diff-f58493e07185c2088e765242d2d67f23407614b295cb9919452821b21123c453R12-R15. It gives a nice consistent setup for guides for different enviroments and how Kuadrant can be installed and decouples from the development workflow 👍